platforms.linaro.org/Enterprise/README.md

```diff
@@ -1,2 +1,10 @@
 # Enterprise Reference Platforms
 
+The enterprise reference platform is targeted to accelerate the high powered ARMv8 servers in both datacenter and cloud vertical markets. Unified firmware and Linux kernel are provided for all the supported platforms, simplifying software maintenance and deployment. The high level components such as Openstack, Ceph, Hadoop, and KVM are rigorously validated for each distribution. The enterprise software stack provides two builds based on Debian and Centos. The goal is to help bridge the gap between older technology present in today’s distributions and the latest ARM server optimizations. 
+
```

Comment:
We should also cover that it is following the Server Base Boot Requirements for ARM64 (UEFI, ACPI, etc).